libtiledbsoma/src/utils/logger.cc

```diff
@@ -71,7 +71,12 @@ Logger::Logger() {
             spdlog::level::critical, console_sink->red_bold);
 #endif
     }
-    set_level("INFO");
+    char* env_level = std::getenv("LIBTILEDBSOMA_LOGGING_LEVEL");
```

Comment:
There is a built-in activator for spdlog we could use too.  I have been meaning to expose it in the R package as well.  See https://github.com/eddelbuettel/spdlog#load-log-levels-from-env-variable-or-from-argv

I also find it convenient to turn on R _and_ libtiledbsoma logging:  
https://github.com/single-cell-data/TileDB-SOMA/blob/7cacee1ec3824040879e55fc4bdd7d5df4a56e7b/apis/r/src/rinterface.cpp#L189-L192

Reply:
> I also find it convenient to turn on R and libtiledbsoma logging:

Thanks @eddelbuettel . There's also `tiledbsoma.pytiledbsoma.config_logging("debug")` which is directly callable from Python. And `spdl::set_log_level("DEBUG")` which is directly callable from R. Knowing both of those things, I also do want the option to enable logging _without_ editing source code -- in whichever language.

Thanks for the tip at https://github.com/eddelbuettel/spdlog#load-log-levels-from-env-variable-or-from-argv re `spdlog::cfg::load_env_levels`! TIL :). I do however want the option to set application-specific levels from the environment.

Reply:
Also keep in mind that there are different instances of spdlog involved which is why the C++-side-of-tiledbsoma-R has two enablers, one for C++ (in libtiledbsoma) and one for R + C++ in the R package.  

> I do however want the option to set application-specific levels from the environment.

That is exactly what it does. (Don't get distracted that the reference accidentally goes to my fork.  It is a spdlog upstream feature.)

I used that very feature in the example code I posted on slack when I proposed header-only use of spdlog and fmt as potential fix to occassional build woes.  Anyway ...